daemon/services/api/websocket.go

```
@@ -284,4 +354,4 @@ func (c *WSClient) readPump() {
 		c.setTopics(topics) // nil means "all topics"
 		logger.Debug("WebSocket client updated topic filter: %v", topics)
 	}
-}
+}
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)